lang/cs/TestRunner.proj

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I do mean something like that. I am not sure if my concerns are real/general. Would leave it up to others to chime in.